readme.md

```diff
@@ -1,156 +1,102 @@
 
-[![License: AGPL-3.0-only](https://img.shields.io/badge/license-%20%09AGPL--3.0--only-blue.svg)](https://raw.githubusercontent.com/renovatebot/renovate/main/license)
-[![codecov](https://codecov.io/gh/renovatebot/renovate/branch/main/graph/badge.svg)](https://codecov.io/gh/renovatebot/renovate)
-[![Renovate enabled](https://img.shields.io/badge/renovate-enabled-brightgreen.svg)](https://renovatebot.com/)
-[![Build status](https://github.com/renovatebot/renovate/actions/workflows/build.yml/badge.svg)](https://github.com/renovatebot/renovate/actions/workflows/build.yml)
-![Docker Pulls](https://img.shields.io/docker/pulls/renovate/renovate?color=turquoise)
-[![OpenSSF Scorecard](https://api.securityscorecards.dev/projects/github.com/renovatebot/renovate/badge)](https://securityscorecards.dev/viewer/?uri=github.com/renovatebot/renovate)
+**Features**
+- Delivers updates directly to your code base
+  - Relevant package files are discovered automatically
+  - Pull Requests automatically generated in your repo
+- Provides useful information to help you decide which updates to accept (age, adoption, passing, merge confidence)
+- Highly configurable - Lots of ways to configure it for your custom needs
+- Largest collection of languages and platforms (listed below)
+- Connects with private repositories and package registries
 
-## Why Use Renovate?
+**Languages**<br>
+Renovate can provide updates for nearly all languages, platforms and registries including: npm, Java, Python, .NET, Scala, Ruby, Go, Docker-hub and more.
+Supports over 90 different package managers.
 
-- Get automated Pull Requests to update your dependencies
-- Reduce noise by running Renovate on a schedule, for example:
-  - on weekends
-  - outside of working hours
-  - each week
-  - each month
-- Relevant package files are discovered automatically
-- Supports monorepo architectures with workspaces with no extra configuration
-- Bot behavior is customizable via configuration files (config as code)
-- Use ESLint-like shared config presets for ease of use and simplifying configuration (JSON format only)
-- Lock files are supported and updated in the same commit, including immediately resolving conflicts whenever PRs are merged
-- Get replacement PRs to migrate from a deprecated dependency to the community suggested replacement, works with _most_ managers, see [issue 14149](https://github.com/renovatebot/renovate/issues/14149) for exceptions
-- Open source (installable via npm/Yarn or Docker Hub) so can be self-hosted or used via the Mend Renovate App
+**Platforms**<br>
+Renovate updates code repositories on the following platforms: GitHub, GitLab, Bitbucket, AzureDevOps, AWS Code Commit, Gitea, Forgejo, Gerrit (experimental)
 
-## Supported Platforms
+#  Ways to Run Renovate
 
-Renovate works on these platforms:
+The easiest way to run Renovate is to use a job scheduling system that regularly runs Renovate on all enabled repositories and responds with priority to user activity.
```

Comment:
I assume you mean CE/EE, but that's not actually the easiest. The easiest is to not run it yourself. Second easiest is the CLI, third easiest/last is CE/EE. So I'm not sure you should use "easiest" if you're trying to encourage CE/EE

Reply:
Changed to -
The most effective way to run Renovate is to use an automated job scheduling system that regularly runs Renovate on all enabled repositories and responds with priority to user activity. Mend.io offers cloud-hosted and self-hosed solutions.